ecs_pod_creator.go

```diff
@@ -867,6 +882,61 @@ func (s *SecretOptions) hash() string {
 	return h.Sum()
 }
 
+// LogConfiguration represents the configuration for a container's logging.
+type LogConfiguration struct {
+	// LogDriver is the logging driver to use.
+	LogDriver *string
+	// Options are the logging driver options.
+	Options map[string]*string
```

Comment:
Do the values need to be string pointers? Can we instead just convert them to string pointers when passing them to AWS?

Reply:
Sure, changed it to regular strings and added that logic into `exportLogConfiguration`